google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -292,7 +312,13 @@ public static EnhancedBigtableStub create(EnhancedBigtableStubSettings settings)
   private UnaryCallable<RowMutation, Void> createMutateRowCallable() {
     MutateRowCallable userFacing = new MutateRowCallable(stub.mutateRowCallable(), requestContext);
 
-    return userFacing.withDefaultCallContext(clientContext.getDefaultCallContext());
+    UnaryCallable<RowMutation, Void> traced =
```

Comment:
This snippet is duplicated quite a bit.  Is there a way to reduce the duplication?

Reply:
Not sure how... its instantiating the middleware for tracing. Did you have something in mind?

Reply:
Fixed as discussed offline.